mne/bem.py

```diff
@@ -1355,3 +1357,260 @@ def write_bem_solution(fname, bem):
+    if convertT1:
+        if not op.isfile('T1.mgz'):
+            raise RuntimeError("Both T1 mgz and T1 COR volumes missing.")
+        os.makedirs('T1')
+        cmd = ['mri_convert', 'T1.mgz', 'T1']
+        run_subprocess(cmd, env=env, stdout=sys.stdout)
+    else:
+        logger.info("T1 volume is already in COR format")
+    logger.info("\n---- Converting brain volume into COR format ----")
+    if convertbrain:
+        if not op.isfile('brain.mgz'):
+            raise RuntimeError("Both brain mgz and brain COR volumes missing.")
+        os.makedirs('brain')
+        cmd = ['mri_convert', 'brain.mgz', 'brain']
+        run_subprocess(cmd, env=env, stdout=sys.stdout)
+    else:
+        logger.info("Brain volume is already in COR format")
+    # Finally ready to go
+    logger.info("\n---- Creating the BEM surfaces ----")
+    cmd = ['mri_make_bem_surfaces', subject]
+    run_subprocess(cmd, env=env, stdout=sys.stdout)
+    logger.info("\n---- Converting the tri files into surf files ----")
+    os.chdir(bem_dir)
+    if not op.exists('flash'):
+        os.makedirs('flash')
+    os.chdir('flash')
+    surfs = ['inner_skull', 'outer_skull', 'outer_skin']
+    for surf in surfs:
+        shutil.move(op.join(bem_dir, surf + '.tri'), surf + '.tri')
+        cmd = ['mne_convert_surface', '--tri', surf + '.tri', '--surfout',
+               surf + '.surf', '--swap', '--mghmri',
+               op.join(subjects_dir, subject,
+                       'mri/flash/parameter_maps/flash5_reg.mgz')]
+        run_subprocess(cmd, env=env, stdout=sys.stdout)
+    # Cleanup section
+    logger.info("\n---- Cleaning up ----")
+    os.chdir(bem_dir)
+    os.remove('inner_skull_tmp.tri')
+    os.chdir(mri_dir)
+    if convertT1:
+        shutil.rmtree('T1')
+        logger.info("Deleted the T1 COR volume")
+    if convertbrain:
+        shutil.rmtree('brain')
+        logger.info("Deleted the brain COR volume")
+    shutil.rmtree('flash5')
+    logger.info("Deleted the flash5 COR volume")
+    logger.info("\nThank you for waiting.\nThe BEM triangulations for this "
+                "subject are now available at:\n%s.\nWe hope the BEM meshes "
+                "created will facilitate your MEG and EEG data analyses."
+                % op.join(bem_dir, 'flash'))
+    # Show computed BEM surfaces
+    if show:
+        os.chdir(op.join(bem_dir, 'flash'))
+        fnames = ['outer_skin.surf', 'outer_skull.surf', 'inner_skull.surf']
+        head_col = (0.95, 0.83, 0.83)  # light pink
+        skull_col = (0.91, 0.89, 0.67)
+        brain_col = (0.67, 0.89, 0.91)  # light blue
+        colors = [head_col, skull_col, brain_col]
+        from mayavi import mlab
+        mlab.clf()
+        for fname, c in zip(fnames, colors):
+            points, faces = read_surface(fname)
+            mlab.triangular_mesh(points[:, 0], points[:, 1], points[:, 2],
+                                 faces, color=c, opacity=0.3)
+        mlab.show()
```

Comment:
No. Add a --overwrite option that checks if sym links should be forced. Otherwise skip the sym links 

> On 4 oct. 2015, at 23:15, lorenzo-desantis <notifications@github.com> wrote:
> 
> In mne/bem.py:
> 
> > +                % op.join(bem_dir, 'flash'))
> > +    # Show computed BEM surfaces
> > +    if show:
> > +        os.chdir(op.join(bem_dir, 'flash'))
> > +        fnames = ['outer_skin.surf', 'outer_skull.surf', 'inner_skull.surf']
> > +        head_col = (0.95, 0.83, 0.83)  # light pink
> > +        skull_col = (0.91, 0.89, 0.67)
> > +        brain_col = (0.67, 0.89, 0.91)  # light blue
> > +        colors = [head_col, skull_col, brain_col]
> > +        from mayavi import mlab
> > +        mlab.clf()
> > +        for fname, c in zip(fnames, colors):
> > +            points, faces = read_surface(fname)
> > +            mlab.triangular_mesh(points[:, 0], points[:, 1], points[:, 2],
> > +                                 faces, color=c, opacity=0.3)
> > +        mlab.show()
> In case the .surf files exists already into the bem folder, should they be overwritten by default?
> 
> —
> Reply to this email directly or view it on GitHub.
>